modules/@apostrophecms/piece-type/index.js

```diff
@@ -1060,6 +1060,84 @@ module.exports = {
             });
           }
         }
+      },
+
+      localize: {
+        usage: 'Add draft version documents for each locale when a module has the "localized" option.' +
+        '\nExample: node app [moduleName]:localize',
+        async task() {
+          if (self.options.localized) {
+            console.log('Adding draft version to documents');
+            const locales = Object.keys(self.apos.i18n.locales);
+
+            await self.apos.migration.eachDoc({ type: self.name }, async doc => {
+              await self.apos.doc.db.removeOne({ _id: doc._id });
+
+              if (doc.aposDocId && !doc._id.endsWith('published') && !doc._id.endsWith('draft')) {
+                for (const locale of locales) {
+                  const lastPublishedAt = new Date();
+                  const newDraft = {
+                    ...doc,
+                    aposLocale: `${locale}:draft`,
+                    aposMode: 'draft',
+                    aposDocId: doc._id,
+                    _id: `${doc.aposDocId}:${locale}:draft`,
+                    lastPublishedAt
+                  };
+                  const newPublished = {
+                    ...doc,
+                    aposLocale: `${locale}:published`,
+                    aposMode: 'published',
+                    aposDocId: doc._id,
+                    _id: `${doc.aposDocId}:${locale}:published`,
+                    lastPublishedAt
+                  };
+                  await self.apos.doc.db.insertOne(newDraft);
```

Comment:
You can use `insertMany` here for a lil' speed boost, array of two. Actually, you can do an array including all of them across all of the locales and make just one `insertMany` call.